src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -97,21 +114,95 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     toggle: togglePersonalisedSchedule,
   } = useShowHide(false);
 
+  const liveAndFutureEvents = useMemo(
+    () => relatedVenueEvents.filter(isEventLiveOrFuture),
+    [relatedVenueEvents]
+  );
```

Comment:
I think would be better to add here a `.map` prepareForSchedule, what do you think?

Reply:
added map there and removed in cases below 58b5cf2